deployment/debian-package-x64/clean.sh

```diff
@@ -2,6 +2,29 @@
 
 source ../common.build.sh
 
-VERSION=`get_version ../..`
+keep_artifacts="${1}"
 
-clean_jellyfin ../.. Release `pwd`/dist/jellyfin_${VERSION}
+WORKDIR="$( pwd )"
+VERSION="$( grep '^jellyfin' ${WORKDIR}/pkg-src/changelog | head -1 | awk -F '[()]' '{ print $2 }' )"
```

Comment:
what is this doing here if it isn't used?

Reply:
Indeed.